Buildings/Examples/ScalableBenchmarks/ZoneScaling/MixedAir/package.mo

```diff
@@ -0,0 +1,19 @@
+within Buildings.Examples.ScalableBenchmarks.ZoneScaling;
+package MixedAir "Package with scalable large office buildings modeled with MBL MixedAir"
```

Comment:
Use full words, users may not know what MBL MixedAir is, and if MixedAir gets renamed, this string will be invalid. If you want to link to a specific model, use `<a href=\"modelica://....` in the info section.